charts/argo-cd/Chart.yaml

```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: Support ability to override .global.networkPolicy.create at the component level
```

Comment:
Pls remove old one.

Reply:
Sorry for that! I assumed it was some batched process where all recent changes were included. I've updated it now.